clicommand/bootstrap.go

```diff
@@ -356,6 +357,11 @@ var BootstrapCommand = cli.Command{
 			EnvVar: "BUILDKITE_TRACING_SERVICE_NAME",
 			Value:  "buildkite-agent",
 		},
+		cli.BoolTFlag{
+			Name:   "job-api",
+			Usage:  "Starts the job-api, giving the commands in the job some abilities to intropect and mutate the state of the job.",
+			EnvVar: "BUILDKITE_AGENT_JOB_API",
+		},
```

Comment:
i think we should probably make this `--no-job-api` and make it a `BoolFlag` - the default state should be having the job API enabled, and then allowing people to disable it, rather than having the UX be `--job-api false`, which reads a little confusingly, i think. 

there's a fair bit of prior art [here](https://github.com/buildkite/agent/blob/main/clicommand/agent_start.go#L519-L538)

Reply:
Yeah, that's much better